e2e/workflowTest/upload-confirm-listing-test/bypass-warnings.spec.js

```diff
@@ -32,6 +34,9 @@ describe('listing with no confirm warnings and no errors', () => {
     confirmPage.confirmListing();
     assert.isFalse(confirmPage.warningCheckbox.isDisplayed());
     confirmPage.waitForSuccessfulConfirm();
+    assert.equal(confirmPage.toastContainerTitle.getText(), 'Please stand by');
+    toast.clearAllToast();
+    hooks.waitForSpinnerToDisappear();
     assert.equal(confirmPage.toastContainerTitle.getText(),'Update processing');
```

Comment:
```suggestion
    assert.equal(confirmPage.toastContainerTitle.getText(), 'Update processing');
```

Similar spacing stuff